services/validation/README.md

```diff
@@ -20,6 +92,7 @@ Savings (browser/Savings.sol):
   Compiled with Solidity 0.6.11
   https://solc-bin.ethereum.org/wasm/soljson-v0.6.11+commit.5ef660b1.js
   https://solc-bin.ethereum.org/linux-amd64/solc-linux-amd64-v0.6.11+commit.5ef660b1
+  To re-compile use: `sourcify-validation --prepare-json browser/Savings.sol:Savings Escrow.sol Main.sol Owned.sol provableAPI_0.6.sol Savings.sol metadata.json | solc --standard-json`
```

Comment:
```suggestion
  To re-compile use: `sourcify-validation --prepare-json browser/Savings.sol:Savings Escrow.sol Main.sol Owned.sol provableAPI_0.6.sol Savings.sol metadata.json | solc --standard-json -`
```

Reply:
Or does it also work without?

Reply:
It was tested and only works without the "-".

Reply:
ok, great!